src/ui/components/Widgets/Chart/ChartWidget.tsx

```diff
@@ -485,6 +486,13 @@ export const ChartWidget = (props: ChartWidgetProps) => {
         [context.canEdit, editMode, context.entryDialoguesRef, initName],
     );
 
+    const dashkitConfig = React.useMemo(
```

Comment:
Judging by what lies in the object, I would expect something like dashState instead of dashkitConfig (which is like a common dashboard setting regardless of the state)